docs/blog/2018-2-7-jam-out-your-blog/index.md

```diff
@@ -0,0 +1,158 @@
+
+_**Yes, there’s more.**_ 
+
+Because Prose was built for Jekyll, it recognizes any headmatter you add to your .md files.  This will give developers the ability to build queries and replicate familiar features from other CMS platforms. Say hello to publish states, publish date, article author and more. 
+
+I should also mention that Prose is an open source project that is available for download if you wish to self-host. Check out the [documentation on Github.](https://github.com/prose/prose)
+
+## Get Lean and Get on Netlify
+
+Let’s summarize where we are and why [Netlify](https://www.netlify.com/) will take care of the rest. 
+
+~~~[ ] Lean - simplify the tools and services use to a bare minimum
+
+[ ] Inexpensive Hosting - again, to maximize value and keep personal costs down
+
+[ ] Continuous Deployment - the site needs to always be up and running and current
+
+[x] Speed - this includes development, website performance, and deployment
+
+[x] Development Experience - I want to use React and Webpack 
+
+[x] Maintainability - ease with which anyone can create, update and delete posts
+
+[x] Scalability - need to make sure that performance is not hindered by a growing needs
+
+[x] Revision History - for accountability and allowing for versioning 
+
+[x] NO WordPress - similarly, I don’t want to deal with updates, themes, hacks...etc
+~~~
+
+When it comes to continuous deployment and inexpensive hosting, using Netlify is, as Josh mentions “ a no brainer.”  Check out [Netlify’s features here](https://www.netlify.com/features/) for a more in-depth look.
+
+With streamlining in mind, a workflow doesn’t get any leaner than working within the confines of Github and Netlify. Think about the magic:  
+
+1.  Use Gatsby to create your website.
+2.  Set up a GitHub repository to track changes and version history.
+3.  Once ready, deploy your repository to Netlify.
+4.  Use Prose (which accesses your GitHub), to create, edit and commit posts.
+5.  Netlify sees the update to the repository and re-renders the website.
+
+With this process in place, content creators do not need to understand Markdown, React, Git or Netlify. They can simply log into Prose and start writing. All the developer has to do is share the repository. 
+
+_**It just works.**_ 🤙🏼
+
+<div style="width:100%;height:0;padding-bottom:55%;position:relative;"><iframe src="https://giphy.com/embed/3oFzmqDNycyhUcrjTW"  width="100%" height="100%" style="position:absolute" frameBorder="0" class="giphy-embed"  allowFullScreen></iframe></div><p><a href="https://giphy.com/gifs/nba-celebrity-3oFzmqDNycyhUcrjTW">via GIPHY</a></p>
+
+
+
+## Concluding Thoughts
+
+Let me be clear: there is definitely a need for services such as Contentful. I just feel as though this process removes much of the overhead that is overkill for most projects. 
```

Comment:
Not language we'd want in an official blog post -- can you remove this para? Your exhortation should be sufficient encouragement to folks :)

Reply:
I'll remove it in my next PR.